README.md

```diff
@@ -39,14 +39,21 @@ Sign up to [get started for free](https://app.revenuecat.com/signup).
 🔀 | [Integrations](https://www.revenuecat.com/integrations) - over a dozen integrations to easily send purchase data where you need it
 💯 | Well maintained - [frequent releases](https://github.com/RevenueCat/purchases-ios/releases)
 📮 | Great support - [Help Center](https://community.revenuecat.com)
+🤩 | Awesome [new features](https://trello.com/b/RZRnWRbI/revenuecat-product-roadmap)  
```

Comment:
Is this public? At the very least looks like it makes you log-in to see it.

Reply:
Oh yeah:
<img width="549" alt="Screenshot 2022-08-19 at 13 35 15" src="https://user-images.githubusercontent.com/685609/185703070-b4e88d05-75ec-496b-b89a-a4f9f4215a13.png">

Reply:
I can take this off here and purchases-android. Unless it's meant to be enticing...?

Reply:
LOL that's... ehem... _dated_. 
As in, it's been closed for like 2 years. We should remove it

Reply:
Removed.